shipit.yml

```diff
@@ -12,3 +12,12 @@ deploy:
 ci:
   allow_failures:
     - "yarn-audit"
+
+tasks:
+  renew_cert:
+    action: "Renew SSL Certificate"
+    description: "Send a request via certbot to issue an SSL certificate"
+    steps:
+      - export ISSUE_CERT=true
+      - make install:
```

Comment:
Nope, the first `make install` is part of the deploy target, so that's what shipit will run when deploying. The second `make install` is part of a task, this task is shown on the shipit dashboard and can be run manually. When run manually, it will trigger a deploy with `--set certbot.manualRun=true` which sends a request to renew the cert.